README.md

```diff
@@ -32,7 +32,7 @@ The Zoo also provides analytics about how dependency usage and its versions evol
 ### Initial setup
 
 - Create a database: `$ make migrate`
-- Create a superuser: `$ make superuser`
+- Create a superuser and fake data: `$ make fake`
```

Comment:
So, from what I see `make superuser` will still work, it's just that if not used we'll create one with `admin:password` credentials, right?

Reply:
That’s the idea.

Reply:
Great